### PR TITLE
Feature/styling

### PR DESCRIPTION
**Pull request by alia-peterson**
## *Purpose:*
*What does this merge do? Check all that apply.*
- [ ] Adds a new feature
- [x] Improves an existing feature
- [ ] Fixes a bug
- [ ] Cleans up (logs, tests, etc.)

## *Description: What does this PR do?*
Updated styling to make a more cohesive website

## *What are the relevant tickets?*
NA

## *Bugs:*
Some bugs for when a page is refreshed, should be handled in another PR

## Checklist:
- [x] I have performed a self-review of my own code
- [x] I have run Lighthouse/Wave checks for accessibility
- [ ] New and existing unit tests pass locally with my change

### CODE REVIEW

File: components/CircleChart.js [hunk 1]
```diff
@@ -1,55 +1,29 @@
...
-          "#3f218c",
-          "#ebfc05",
-          "#42e6d0",
-          "#bb0935",
-          "#5d9b35",
-          // '#ffa500',
-          // '#ff2500'
-        ],
+        data: chartData,
+        backgroundColor: activityColors,
```
Comment (camaragon) [inline components/CircleChart.js:17]: I did this on my current branch but a little differently, but I like your way so we should keep this commit.

File: utilities/chartColors.js [hunk 2]
```diff
@@ -0,0 +1,20 @@
+const colors = {
```
Comment (srslie) [inline utilities/chartColors.js:1]: Nice, hardcoding it is definitely a great option with a small dataset like we have.

**Review by srslie — APPROVED** [review]
Thanks for your attention to detail.

File: components/NavBar.js [hunk 3]
```diff
@@ -74,13 +64,13 @@ export default function NavBar() {
 
   return (
     <nav className={styles.navbar}>
-        <Link href='/' className={styles.homeLogo}>
```
Comment (camaragon) [inline components/NavBar.js:77]: These odd double indents! Nice catch

File: utilities/chartColors.js (hunk 2 above)
Comment (camaragon) [inline utilities/chartColors.js:1]: I will add unit testing to this file!

**Review by camaragon — APPROVED** [review]
This all makes the application look so much better!

**Review by lbmerchant93 — APPROVED** [review]
Looks great! Good job on adding the styling to the chartjs elements, condensing those scss variables, and having the same colors across all charts!

**Pull request merged**